src/monitor/pgautofailover.sql

```diff
@@ -160,7 +160,7 @@ AS 'MODULE_PATHNAME', $$node_active$$;
 
 grant execute on function
       pgautofailover.node_active(text,text,int,int,int,
-                          pgautofailover.replication_state,bool,bigint,text)
+                          pgautofailover.replication_state,bool,text,text)
```

Comment:
So this should be `pg_lsn` so that Postgres does the text to LSN transformation for us... can we make that work?